src/test/transaction_tests.cpp

```diff
@@ -784,6 +784,19 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     reason.clear();
     BOOST_CHECK(!IsStandardTx(CTransaction(t), reason));
     BOOST_CHECK_EQUAL(reason, "multi-op-return");
+
+    // Check large scriptSig (non-standard if size is >1650 bytes)
+    t.vout.resize(1);
+    t.vout[0].nValue = 90*CENT;
```

Comment:
this value seems like a magic number unrelated to the actual test. Could just make it MAX_MONEY since we don't want to make it dusty.

Reply:
Sure, done.